web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -53,6 +55,15 @@ const RegistryLayoutHeader: React.FC = () => {
   const { showProjects, showCreditClasses, isIssuer } = useProfileItems({});
   const menuItems = useMemo(() => getMenuItems(pathname, _), [pathname, _]);
   const onProfileClick = useOnProfileClick();
+
+  const { data: paymentMethodData } = useQuery(
+    getPaymentMethodsQuery({
+      enabled: !!isConnected,
```

Comment:
this isn't right, we should check for payment methods if there's an active account id (not necessarily a connected wallet address), check how it's done in my PR: https://github.com/regen-network/regen-web/blob/ad280a358c6b2482922816dbb8ea07eea95b9de2/web-marketplace/src/pages/BuyCredits/BuyCredits.Form.tsx#L127

Reply:
Ah ok, I saw that but it seems I didn’t fully get it. Do you mean we should use `useAuth()` in this case too?

Reply:
yes

Reply:
Thanks